library.sh

```diff
@@ -770,8 +770,8 @@ function __go_update_deps_for_module() {
     remove_broken_symlinks ./vendor
   fi
 
-  group "Updating licenses"
-  update_licenses third_party/VENDOR-LICENSE "./..."
+  group "Checking licenses"
+  check_licenses
```

Comment:
Maybe we should actively delete those `LICENSE` files, instead of relying on of every project contributors to do it manually?

Reply:
yeah sounds good

Reply:
actually - that might break builds

Reply:
because we need to stop the symlink to third party licenses

Reply:
What might break builds? The removal of the LICENSE files?

Reply:
I like the idea @dprotaso proposed, to add the deletion of those licenses in the next release.